layouts/blog/single.email.html

```
@@ -60,7 +60,7 @@ <h1>{{ .Title }}</h1>
       <a href="{{ .Site.BaseURL }}">adrianmoreno.info</a>.
     </p>
     <p>
-      <a href="{{ "{{{RESEND_UNSUBSCRIBE_URL}}}" | safeHTML }}">Unsubscribe</a>
+      <a href="{{ "{{{RESEND_UNSUBSCRIBE_URL}}}" | safeURL }}">Unsubscribe</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)